mcs/class/corlib/Test/System/ArrayTest.cs

```diff
@@ -1002,8 +1002,7 @@ public void CreateInstanceVoid ()
 		int[] myLengthArray = new int[1] { 3 };
 		int[] myBoundArray = new int[1] { Int32.MinValue };
 		Array myExtremeArray=Array.CreateInstance ( typeof(String), myLengthArray, myBoundArray );
-		Assert.AreEqual (Int32.MaxValue, ((IList)myExtremeArray).IndexOf (42), "AD04");
-
+		Assert.AreEqual (-1, ((IList)myExtremeArray).IndexOf (42), "AD04");
```

Comment:
If the CoreRT implementation doesn't do this then we should open a bug with them.